macros/CompatEdge.ejs

```diff
@@ -190,6 +195,36 @@ if (!release) {
         release = {
             version: str20OrEarlier
         };
+    } else if (Math.floor(Number(edgeHTMLVersion)) >= 10049) {
+        if (Math.floor(Number(edgeHTMLVersion)) >= 10240) {
+            // attempt to handle build numbers
+            var HTMLVersion = 0;
+            var edgeVersion = 0;
+            var BuildNumber = 0;
+            var releaseFound = false;
+            for (var key of releases) {
+                if (releases.hasOwnProperty(key)) {
+                    edgeVersion = Math.floor(Number(releases[key].version));
+                    HTMLVersion = Math.floor(Number(key.split('.')[0]));
```

Comment:
I don't understand why we are doing things with edgeVersion and HTMLVersion. They don't seem to be used.